src/RegistrationController.js

```diff
@@ -70,14 +70,26 @@ function (
       // setup schema
       var Schema = RegistrationSchema.extend({
         settings: this.options.settings,
-        url: this.getRegistrationApiUrl()+'/form'
+        url: this.getRegistrationPolicyApi()+'/form'
       });
       var schema = new Schema();
       this.state.set('schema', schema);
     },
     getRegistrationApiUrl: function() {
-      var clientId = this.options.settings.get('clientId');
-      return this.options.settings.get('baseUrl')+'/api/v1/registration/'+clientId;
+      // default policyId
+      var defaultPolicyId = this.settings.get('defaultPolicyId');
+      // org policyId
+      var orgPolicyId = this.options.settings.get('policyId');
+      var apiUrl = defaultPolicyId ? this.getRegistrationPolicyApi(defaultPolicyId) :
+        this.getRegistrationPolicyApi(orgPolicyId);
+      return apiUrl;
+    },
+    getRegistrationPolicyApi: function(policyId) {
+      if (policyId) {
+        return this.options.settings.get('baseUrl') + '/api/v1/registration/' + policyId;
+      } else {
+        return this.options.settings.get('baseUrl') + '/api/v1/registration';
```

Comment:
why do we need to return `this.options.settings.get('baseUrl') + '/api/v1/registration';` in the else ?

Reply:
fixed